targets/xbmc

```diff
@@ -1,35 +0,0 @@
-#!/bin/sh -e
```

Comment:
Wrap [the common target](https://github.com/dnschneid/crouton/blob/master/targets/common#L30) lines 30 through 34 in a `if [ -n "$DESCRIPTION" ]` block :)